pkg/nb/types.go

```diff
@@ -649,7 +683,50 @@ func IntToHumanBytes(bi int64) string {
 // UInt64ToBigInt convert uint64 based value to BigInt value
 func UInt64ToBigInt(value uint64) BigInt {
 	return BigInt{
-		Peta: int64(value / petaInBytes),
-		N:    int64(value % petaInBytes),
+		Peta: int64(value / PETABYTE),
+		N:    int64(value % PETABYTE),
+	}
+}
+
+// HumanBytesToValueAndUnit convert human bytes to value and unit. 10G -> value: 10, unit GIGABYTE
+func HumanBytesToValueAndUnit(str string) (int, string, error) {
```

Comment:
I think you should use kubernetes `resource.ParseQuantity` instead of parsing yourself. 
see rook implementation: 

https://github.com/rook/rook/blob/fa3ab50eff4795d357041805f0d5a873eb2f4fcc/pkg/operator/ceph/object/bucket/provisioner.go#L580-L591

Reply:
resource.ParseQuantity returns value not in bytes. for example 1M = 1000000 and not 1048576, however the noobaa-core parses the maxSize to bytes value.

Reply:
in kuberentes there is a difference between `1M` and `1Mi`, this is probably why you don't get what you expect
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits

Reply:
Since the noobaa-core translates the value into bytes only, do you still want to use this method?
Or do you prefer to implement the same logic in the core as well?